js/screens/EditProfile/EditProfileContainer.js

```diff
@@ -1,32 +1,134 @@
 import React, { Component } from "react";
-import PropTypes from "prop-types";
 import EditProfile from "./EditProfile";
-import { View, Text } from "react-native";
-import { styles } from "./styles";
+import { updateUserProfile } from "../../helpers/firebaseAuth";
+import { updateProfilePic } from "../../helpers/uploadProfilePicHelper";
+import { updateWorkStyle } from "../../helpers/firebaseStore";
+import ImagePicker from "react-native-image-picker";
 
 class EditProfileContainer extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      user: ""
```

Comment:
Is this component ever used to edit a profile after this information has been saved?

If so, you need a way to get that previously saved data into your form so it can be edited. Otherwise, this component will only be good for filling in a completely empty profile.